src/Sarif/ConsoleLogger.cs

```diff
@@ -219,9 +251,10 @@ private static void ValidateKindAndLevel(ResultKind kind, FailureLevel level)
                 location = region.FormatForVisualStudio();
             }
 
-            string result = (path != null ? (path + location + ": ") : "") +
-                   issueType + (!string.IsNullOrEmpty(ruleId) ? " " : "") +
-                   (!string.IsNullOrEmpty(ruleId) ? (ruleId + ": ") : "") +
+            string result = 
```

Comment:
>result [](start = 19, length = 6)

While we're in here, suggested variable name: `messageText` (esp. since method is `GetMessageText`).